imagersite/imagersite/tests.py

```diff
@@ -49,35 +169,141 @@ def test_register_user(self):
 class ImagesRoutesTests(TestCase):
     """Class for testing views."""
```

Comment:
All of the tests related to the `imager_images` app should be in the test file for that app. Not in this generic tests file.